script.js

```diff
@@ -105,6 +210,10 @@ $(document).on('keypress', function (e) {
   if (e.which == 13) {
     spin(2);
   }
+  // listen on h key and hide and unhide
+  if (e.which == 104) {
+    menuItem.hidden = !menuItem.hidden;
+  }

```

Comment:
Can we make it hidden by default? This way, we don't expose it to customers when turning on the wheel

Reply:
Resolved